go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -181,8 +176,6 @@ func (fe *FileEntry) open(cnf *Mycnf, readOnly bool) (*os.File, error) {
 // ExecuteBackup runs a backup based on given params. This could be a full or incremental backup.
 // The function returns a boolean that indicates if the backup is usable, and an overall error.
 func (be *BuiltinBackupEngine) ExecuteBackup(ctx context.Context, params BackupParams, bh backupstorage.BackupHandle) (bool, error) {
-	params.Logger.Infof("Hook: %v, Compress: %v", backupStorageHook, backupStorageCompress)
```

Comment:
Might be informative to instead log the `params` but IMO fine to just remove too.

Reply:
I've added a log line that prints more info.
```
I0111 13:48:44.086977   11714 builtinbackupengine.go:179] Executing Backup at 2023-01-11 13:48:39.006292 -0800 PST m=+2.432865209 for keyspace/shard ks/0 on tablet vtbackup-2077360256, concurrency: 4, compress: true, incrementalFromPos:
```